beetmoverscript/test/test_script.py

```diff
@@ -131,6 +131,7 @@ def test_move_beet(event_loop):
     context = Context()
     context.config = get_fake_valid_config()
     context.task = get_fake_valid_task()
+    locale = "sample-locale"
```

Comment:
in-Tree stuff uses "ab-CD" as a locale code for testing/invalid purposes, maybe use that here, just for clarity?